src/main/java/embl/ebi/variation/eva/pipeline/gene/FeatureCoordinates.java

```diff
@@ -32,21 +32,21 @@
     public FeatureCoordinates() {
     }
 
-    public FeatureCoordinates(String featureId, String name, String feature, String chromosome, int start, int end) {
-        this.featureId = featureId;
+    public FeatureCoordinates(String id, String name, String feature, String chromosome, int start, int end) {
+        this.id = id;
```

Comment:
Is this field automatically serialized as "_id" in the database? Otherwise an extra unique index is required in the collection.